### DISCUSSION TITLE
请问下 替换翻译 这个功能是做什么用的？

### DISCUSSION THREAD

**Discussion by TnZzZHlp** [STranslate/STranslate#124] [Q&A] [answered]
感谢作者的付出，搞了半天没搞懂这个功能的用法

**Comment by ZGGSONG** [comment, marked as answer]
就比如说你正在和一个外国人聊天或者给开源项目提pr，你正常写汉字就行，写完全选，通过热键触发替换翻译，走你配置的服务和目标语种，这样你的文本翻译成指定语言了，一键就搞定了。

当然软件还同时支持插入文本，就是你滑词翻译等待翻译结果，每条翻译结果旁边会有一个插入图标，你可以选择质量最高的的一条插入文本，实现替换

**Reply by TnZzZHlp** [reply]
噢，懂了，感谢解答
